brayns/io/MorphologyLoader.cpp

```diff
@@ -76,6 +76,7 @@ class MorphologyLoader::Impl
      */
     bool importMorphology(const servus::URI& source, Model& model,
```

Comment:
Throw exception on failure and return structure.

Reply:
In the end, the only thing I need to return is a soma position, meaning I don't need a structure there.